src/status_im/chat/models/mentions.cljs

```diff
@@ -193,7 +193,8 @@
    [alias name nickname]))
 
 (defn add-searchable-phrases-to-contact
-  [{:keys [alias name added? blocked? identicon public-key nickname ens-verified]} community-chat?]
+  [{:keys [alias name added? blocked? identicon public-key nickname ens-verified images]}
```

Comment:
why are we modifying files in status_im, can this not code not be moved to status_im 2 and used from there?

Reply:
consider making a follow up to move these files to the correct new location